Makefile

```diff
@@ -75,10 +78,16 @@ ARKOUDA_MAIN_SOURCE := $(ARKOUDA_SOURCE_DIR)/$(ARKOUDA_MAIN_MODULE).chpl
 $(ARKOUDA_MAIN_MODULE): $(ARKOUDA_SOURCES) $(ARKOUDA_MAKEFILES)
 	$(CHPL) $(CHPL_DEBUG_FLAGS) $(CHPL_FLAGS) $(ARKOUDA_MAIN_SOURCE) -o $@
 
-CLEAN_TARGETS += arkouda-clean
-.PHONY: arkouda-clean
-arkouda-clean:
-	$(RM) $(ARKOUDA_MAIN_MODULE) $(ARKOUDA_MAIN_MODULE)_real
```

Comment:
I'm probably missing something obvious, but I don't understand why these lines went away.

Reply:
Ah, I reviewed too quickly... you've restored them now.